lib/primitives/base-element.js

```diff
@@ -116,10 +137,22 @@ module.exports = function(registry) {
     },
 
     fromRefract: function(doc) {
-      this.meta = doc.meta;
       this.attributes = doc.attributes;
       this.content = doc.content;
 
+      if (doc.meta) {
+        Object.keys(doc.meta).forEach(function (key) {
+          var value = doc.meta[key];
+
+          // TODO: Warning - this may raise a false positive :-(
```

Comment:
@danielgtaylor Is this saying:

If there is an `element` property and either `meta` or `attributes`, consider it as a Refract object?